app/subscriber/src/features/content/list-view/interfaces/IContentListFilter.ts

```diff
@@ -7,6 +7,7 @@ export interface IContentListFilter {
   boldKeywords?: boolean;
   contentTypes: ContentTypeName[];
   excludeSourceIds?: number[];
+  excludeProductIds?: number[];
```

Comment:
Remember we are moving away from this to use Elasticsearch queries instead.

Reply:
Thought of this as I was falling asleep last night, changing this!

Reply:
Adjusted to use elastic queries, also fixed a bug with the date persisting when navigating back (much similar to how it would lose state on other filters) @Fosol